### ISSUE TITLE
no images in the frontend (404) after converting and activating

### ISSUE THREAD

**Issue by bjbommer** [FriendsOfShopware/FroshWebP#129]
### PHP Version

7.4

### Shopware Version

5.7.16

### Actual behaviour

The command ./bin/console frosh:webp:generate runs on our dev for 2h.

STACK: 5906
OFFSET: 0
   95/5906 [>---------------------------]   1%
media/image/betriebsurlaub.jpg => File not found at path: media/image/54/96/d7/betriebsurlaub.jpg
 1592/5906 [=======>--------------------]  26%
media/image/5in1_T-Moulding.tif => Could not load image

media/image/5in1_T-Moulding_Carpet.tif => Could not load image
 2156/5906 [==========>-----------------]  36%
media/image/08-BCL50_montage_cmyk.tif => Could not load image
 2161/5906 [==========>-----------------]  36%
media/image/07-BCL50_montage_cmyk.tif => Could not load image
 2362/5906 [===========>----------------]  39%
media/image/VXX18-40050_TS.tif => Could not load image
 5544/5906 [==========================>-]  93%
media/image/9062302001.tif => Could not load image
media/image/9062302001mt6gfmHmwJgKg.tif => Could not load image
 5562/5906 [==========================>-]  94%
media/image/9062402001.tif => Could not load image
 5597/5906 [==========================>-]  94%
media/image/9062182001_Ecke-hamburg.tif => Could not load image
 5890/5906 [===========================>]  99%Killed

After the converting, no image is shown in the Frontend (404) and on the file system i can't find a image with .webp. 
Where will be the converted images be saved? I can't find an error-log conserning to the converting. 

### Expected behaviour

The converter should convert the images and save it on the server. 
The Frontend should display the .webp images. 

### Steps to Reproduce?

i run the command  ./bin/console frosh:webp:generate in the Terminal
after converting i activate the plugin options for webp in the backend and cleared all the caches.

**Comment by shyim** [comment]
also run sw:thumbnail:generate -f?

**Comment by bjbommer** [comment]
no, i hadn't. i thougth all images are generated by uploading over shopware media. 
i run the command. After 7h it was done. After 100% i get some MySQL Errors, but the Server is still running. 

Generating Thumbnails for Album Blog (ID: -11)
 36/36 [============================] 100%
Generating Thumbnails for Album Einkaufswelten (ID: -3)
 104/104 [============================] 100%
Generating Thumbnails for Album Banner (ID: -2)
    0 [>---------------------------]
Generating Thumbnails for Album Artikel (ID: -1)
 5663/5663 [============================] 100%
Generating Thumbnails for Album Pergo (ID: 5)

In AbstractMySQLDriver.php line 115:                                                                                         
  An exception occurred while executing 'SELECT count(DISTINCT s0_.id) AS sclr_0 FROM s_media s0_ WHERE s0_.albumID  
   = ?' with params [5]:                                                                                                                                                                                                         
  SQLSTATE[HY000]: General error: 2006 MySQL server has gone away                                                                                                                                                              

In Exception.php line 18:                                                          
  SQLSTATE[HY000]: General error: 2006 MySQL server has gone away                                                          

In PDOStatement.php line 117:                                                        
  SQLSTATE[HY000]: General error: 2006 MySQL server has gone away  
                                                                   
sw:thumbnail:generate [--albumid [ALBUMID]] [-f|--force]
